projects/core/src/user/store/reducers/user-orders.reducer.ts

```diff
@@ -1,16 +1,14 @@
-import * as fromUserOrdersAction from '../actions/user-orders.action';
-import * as fromAction from '../actions/index';
```

Comment:
except ` case fromAction.CLEAR_MISCS_DATA:`, all can be removed

Reply:
We have to wait for @dunqan to cover this case, before we can remove it